### ISSUE TITLE
Delayed processing of pre-synaptic spikes

### ISSUE THREAD

**Issue by jlubo** [arbor-sim/arbor#1749]
I need an interface for delayed processing of pre-synaptic spikes. The enhancement should enable a pre-synaptic spike to trigger multiple events in the same post-synaptic neuron at different delay times (in my case, specifically, these are a postsynaptic potential after short delay and an increase in calcium concentration after longer delay, as described [here](https://www.nature.com/articles/s42003-021-01778-y)).

To summarize, it would be great to have something like the `POST_EVENT` block with its `time` argument, just that it should be called at a certain time after a *pre-synaptic* spike.

**Comment by jlubo** [comment]
I don't think so - those synapses would have to communicate somehow (the calcium is needed for changes in synaptic weight). 

Also, even if such a workaround were possible, there would be a lot of redunancy so I guess a proper implementation would be better in terms of performance.

**Comment by jlubo** [comment]
We found that the following workaround does the job. 

We used two connections to the same postsynaptic neuron with different (pseudo) weights `1` and `-1` and different delay times `d0` and `d1`:
```
[arbor.connection((src,"spike_detector1"), "syn_exc_calcium_plasticity", 1, d0), \
arbor.connection((src,"spike_detector1"), "syn_exc_calcium_plasticity", -1, d1)]
```
which can be processed in one and the same synapse mechanism `syn_exc_calcium_plasticity` as follows:
```
NET_RECEIVE(weight) {
	if (weight > 0) {
		: postsynaptic potential evoked by presynaptic spike
	}
	else {
		: increase of calcium amount evoked by presynaptic spike
	}
}
```
Should we close this issue?

**Comment by schmitts** [comment]
I think we can close the issue and think about writing an AEP for allowing additional payloads/slots in connections.

**Comment by thorstenhater** [comment]
As this is related to a PR is this still needed? If not, please dismiss this and the PR.
